inotify.go

```diff
@@ -278,27 +275,6 @@ func (w *Watcher) readEvents() {
 	}
 }
 
-// Certain types of events can be "ignored" and not sent over the Events
-// channel. Such as events marked ignore by the kernel, or MODIFY events
-// against files that do not exist.
-func (e *Event) ignoreLinux(mask uint32) bool {
-	// Ignore anything the inotify API says to ignore
-	if mask&syscall.IN_IGNORED == syscall.IN_IGNORED {
```

Comment:
Are we sure we don't want to ignore events marked with `IN_IGNORE`? The documentation says we should ignore them, but after reading the <a href="http://linux.die.net/man/7/inotify">manpage</a>, it seems like the worst that would happen is we would receive events for files that we already removed from the watch.